tools/wave/docs/rest-api/README.md

```diff
@@ -0,0 +1,54 @@
+# REST API - [WAVE Test Suite](../README.md)
+
+The REST API allows the WAVE server to be integrated into other systems.
+
+## Sessions API <a name="sessions-api"></a>
+
+### Methods
+
+| Name                                           | Description                                          |
+| ---------------------------------------------- | ---------------------------------------------------- |
+| [`create`](./sessions-api/create.md)           | Creates a new test session.                          |
```

Comment:
The documentation for the WAVE test runner is written in markdown, which is split up into multiple files that reference each other. In the example you pointed out, we link to the `create.md` file inside the docs directory. Hence the relative path, starting with `./`.

Reply:
Thanks @jgraham again for your feedback. We solved most of the issues you recommended what is still missing are the integration and unit tests. It would be great if you can guide us on this part. As you can also see, the lastet version causes some CI checks to fail. From the logs we can see that this CI test `ci/tc/tests/test_valid.py F.....` fails. Any idea what could be the problem here?

Reply:
I see. Could we make the default `_wave` or something so that it can't clash with any future spec? We have effectively reserved a few words (`tools`, `resources`, etc.) but since communicating that with the spec community is hard it would be good to avoid reserving more (and this case would be particularly tricky since the existing name won't show up in a directory listing.

Reply:
Thanks @jgraham yes we will change it to `_wave`

Reply:
Now default namespace is `_wave`